### ISSUE TITLE
how to save time series openfoam data to mesh as animation without bvtknodes

### ISSUE THREAD

**Issue by ymg2007** [tkeskita/BVtkNodes#82]
First bvtknodes is amazing!!! 
I followed the cubeflow cut plane example. Then use timeline to show the plane with different time, it worked fine.
![image](https://user-images.githubusercontent.com/24608776/156314188-f58219de-59d5-4e4d-af05-055f654987da.png)
I want to save the time series mesh to an animation in blender for someone using blender without bvtknodes. 
So is there one way to solve the idea?

**Comment by tkeskita** [comment]
Hi, no easy routes to my knowledge. You can save mesh from each time step into separate files, but animation with that is not easy.

**Comment by ymg2007** [comment]
Is there one way to automate save each time step to separate mesh?

**Comment by tkeskita** [comment]
Yes, you can use Python script which first increases frame number, then saves the mesh object. Should be easy, but I don't have exact commands. Have a look in the [Blender Python API documentation](https://docs.blender.org/api/master/).
